form-submission-handler.js

```diff
@@ -44,6 +44,8 @@
     // add form-specific values into the data
     formData.formDataNameOrder = JSON.stringify(fields);
     formData.formGoogleSheetName = form.dataset.sheet || "responses"; // default sheet name
+    formData.formGoogleEmailSubject = form.dataset.subject || "Contact form was submitted"; //default email subject
```

Comment:
is it expected that we set the defaults here, or would we want to have a default like the custom email option below? 

nitpick: the comment at the end matches L46, but I think it could be cleaner code to consider changing the variable names here to those, if those are better descriptions than what we have as variables in the code.

what do you think?